auslib/db.py

```diff
@@ -871,7 +887,7 @@ def _splitColumns(self, columns):
         base_columns = {}
         condition_columns = {}
         for k in columns:
-            if k in itertools.chain(*self.conditions.condition_groups):
+            if k in itertools.chain(*self.conditions.condition_groups.values()):
```

Comment:
Ah, I see what you mean re: global vars now - it's the same issue we see if we have a dict as the default arg in a constructor, and then modify it in the constructor - it will be modified for all consumers. Since this is essentially a static variable, this is probably OK for now. We will need to start copying it if AUSTable starts modifying it at some point.

Good catch on the naming here, I didn't think very hard when switching this!